### DISCUSSION TITLE
テストファームウェアを書き込んだが、すべてのキーが反応しない

### DISCUSSION THREAD

**Discussion by d6ihbtzg** [Yowkees/keyball#487] [Q&A]
キーが反応しません。12p ピンソケットが浮いていますが、これが原因でしょうか？
![20240219_010119](https://github.com/Yowkees/keyball/assets/2555027/e0373399-931b-40a2-bb11-b3fd8eb86226)
電子工作初めての初心者のためどうしたらいいかわからず、ご教示いただけますとありがたいです

**Comment by Ryo624** [comment]
ご連絡ありがとうございます。

12Pピンヘッダがピンソケットに差し込み切れず接続が不安定なことが原因かと思われます。
可能であれば12Ｐピンソケットの奥までピンヘッダを差し込んでみてください。

**Comment by d6ihbtzg** [comment]
ピンソケットに刺さらないのですが、強く差し込む必要がありますでしょうか？

2024年2月19日(月) 10:06 Ryo624 ***@***.***>:

**Comment by Ryo624** [comment]
少し固さはありますがそこまでの強さで差し込む必要はないと思います。
ピンソケット内にはんだが流れ込んでしまいピンヘッダが差し込めない可能性がありますので、新しいものに取り替えた方が確実かと思います。

白銀ラボのウェブショップから購入のご案内ができますので、購入をご希望の場合はお問い合わせフォームからご連絡ください。

**Comment by d6ihbtzg** [comment]
差し込めました。ありがとうございます。それでもキーが反応しないです。原因はどのあたりが考えられますでしょうか？

2024年2月19日(月) 10:22 Ryo624 ***@***.***>:

**Comment by Ryo624** [comment]
基板のはんだ付けを確認させていただきたいので写真の添付をお願いしてもよろしいでしょうか。

**Comment by d6ihbtzg** [comment]
こちらになります。


2024年2月19日(月) 12:07 チャカラピン ***@***.***>:

**Reply by Ryo624** [reply]
申し訳ありません、写真が確認できませんので再度添付をお願いしてもよろしいでしょうか？

**Comment by d6ihbtzg** [comment]
![20240219_120257](https://github.com/Yowkees/keyball/assets/2555027/d8df020b-99d4-43ee-83c9-4fb10de67861)
[](url)

**Reply by d6ihbtzg** [reply]
こちらになります

**Comment by d6ihbtzg** [comment]
GitHubに直接アップしましたのでご確認のほどよろしくお願いします。

2024年2月19日(月) 13:04 Ryo624 ***@***.***>:

**Comment by Ryo624** [comment]
写真の添付ありがとうございます。

写真の赤い丸で示した箇所のはんだが端子に馴染んでおらず浮いているだけに見えます。
こちらのはんだ付けをやり直していただけますでしょうか。
![305789723-d8df020b-99d4-43ee-83c9-4fb10de67861](https://github.com/Yowkees/keyball/assets/143698349/9c5e65b9-b06a-49a4-83e6-2b95f0aef0b8)

**Comment by d6ihbtzg** [comment]
つけ直しましたら、キー反応しました。ありがとうございます

2024年2月19日(月) 16:20 Ryo624 ***@***.***>:

**Comment by Ryo624** [comment]
無事に解決していただきましてありがとうございます。
ぜひ末永くご愛用いただければと思います。
